### ISSUE TITLE
App menu indicator - monochromatic icon?

### ISSUE THREAD

**Issue by m4tx** [ValveSoftware/steam-for-linux#423]
Steam currently uses libappindicator. It's really good thing, but I think... Steam could use it in a better way :) Most of app indicators uses truly monochromatic icon - only white (grey, to be exact) and transparent colors are used. Steam uses its regular icon here. I think that it doesn't look good, especially on Ubuntu (which is the main target of the Steam for Linux), so please, fix it :)
Regards

**Comment by CoolSilver** [comment]
Issue also exists with KDE 5 with Breeze theme

**Comment by Benguin** [comment]
Anybody got a solution that works with XFCE's notification area? I tried placing a modified icon in the places suggested above, but that did nothing. I also tried modifying the icon in steams own files, but steam detects the change and updates to get rid of it (even if the file is marked as read-only.)

**Comment by vikaig** [comment]
Arch Linux, Xfce, default theme, many years have passed, it is still difficult to see the Steam icon in the tray

**Comment by amstelchen** [comment]
A workaround for Xfce is probably [#5307](https://github.com/ValveSoftware/steam-for-linux/issues/5307)

`
Exec=env STEAM_FRAME_FORCE_CLOSE=1 steam`

**Comment by DanielGibson** [comment]
> Issue also exists with KDE 5 with Breeze theme

Ten years later, it still looks like this:
![Image](https://github.com/user-attachments/assets/35ff28d1-6df4-409e-9c6c-928d04080e1b)

Come on Valve, this can't be rocket science?

**Comment by urbenlegend** [comment]
Yeah considering that SteamOS uses KDE as its desktop mode, I am surprised that KDE light mode is so unloved. SteamOS does use dark theme by default, so I guess they avoid the issue.

I think even the default blue steam icon would be better than the white against white.

**Comment by beedell-roke** [comment]
@DanielGibson, that looks more like what [`issues/4715#issuecomment-2558416729`](https://github.com/ValveSoftware/steam-for-linux/issues/4715#issuecomment-2558416729) describes. I'd say that this issue's original request has been fulfilled: that the application icon not be used in the tray. The current icon is monochromatic, but not re-colourable, which the aforementioned ticket tracks.
